Makefile

```diff
@@ -90,20 +90,24 @@ mock:
 	mockgen -source=pkg/handlers/interface.go -destination=pkg/handlers/mock/mock.go
 	mockgen -source=pkg/operator/client/client_interface.go -destination=pkg/operator/client/mock/mock.go
 
+.PHONY: install-gomplate
```

Comment:
i would suggest putting this in the dev dependencies in the README instead like all the other dependencies.

- KOTS: https://github.com/replicatedhq/kots?tab=readme-ov-file#requirements
- EC: https://github.com/replicatedhq/embedded-cluster?tab=readme-ov-file#requirements